src/VS1053.cpp

```diff
@@ -325,4 +325,4 @@ uint16_t VS1053::getDecodedTime() {
 void VS1053::clearDecodedTime() {
     write_register(SCI_DECODE_TIME, 0x00);
     write_register(SCI_DECODE_TIME, 0x00);
-}
\ No newline at end of file
+}
```

Comment:
Are you able to exclude this file from the patchset? 
It looks like it's some accidental change.

Reply:
I'm not sure if this is possible. I did this PR from my presonal branch and this change is there accidentaly.
I will cancel this PR and provide a new one.

Reply:
No need, you can just replace this line from git history or please keep as it is, and I will exclude it while merging.